src/stores/middleware/persistState.ts

```diff
@@ -0,0 +1,18 @@
+import type { Middleware } from "@reduxjs/toolkit";
+
+import type { RootState } from "../store";
```

Comment:
is is common to use relative imports for utilities and keep them separated by a new line?

Reply:
As mentioned above its our auto formatting rules.